src/styles/main.scss

```diff
@@ -358,7 +358,6 @@ body.homepage{
 
 code {
   padding: 2px;
-  white-space: pre-wrap !important;
```

Comment:
Removing `pre-wrap` for all `code` items is dangerous. Let's only do it for code inside a table, please 🙏

Reply:
also how do we feel about only removing it for the first element/column of a table row?

Reply:
> also how do we feel about only removing it for the first element/column of a table row?

Seems like a good idea! +1